bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml

```diff
@@ -23,7 +18,7 @@ metadata:
     capabilities: Seamless Upgrades
     containerImage: icr.io/cpopen/common-service-operator:latest
     createdAt: "2023-11-06T04:32:47Z"
-    description: The IBM Common Service Operator is used to deploy IBM Common Services
+    description: The IBM Cloud Pak foundational services operator is used to deploy IBM foundational services.
```

Comment:
```suggestion
    description: The IBM Cloud Pak Foundational Services operator is used to deploy IBM Foundational Services.
```

Reply:
Will keep as lowercase for consistency with the UI for now.